app/models/concerns/data_anonymizable.rb

```diff
@@ -0,0 +1,27 @@
+module DataAnonymizable
+  extend ActiveSupport::Concern
+
+  UNAVAILABLE = 'Unavailable'
+
+  class_methods do
+    def hash_uuid(original_uuid)
+      UUIDTools::UUID.md5_create(UUIDTools::UUID_DNS_NAMESPACE, { uuid: original_uuid }.to_s).to_s
+    end
+
+    def original_else_blank_value(value)
```

Comment:
We can remove this functionality from the models and move this move the graceful nil-punning (marking nils as 'UNAVAILABLE') to the view (in this case the service that populates the CSV).

Reply:
In which case, the concern may become more specific – just providing a hashing helper for your models.

Reply:
Changed concern name to `Hashable` which now contains a single method, `hash_value` which will return `nil` if the passed value is blank (`nil`) (the onus of handling this is on the client), otherwise return the hashed value instead.